app/templates/components/forms/login-form.hbs

```diff
@@ -1,4 +1,13 @@
 <div class="ui stackable {{unless this.noSocial 'three' 'one'}} column doubling centered grid">
+  <div class="flash-messages ui container">
+    {{#each flashMessages.queue as |flash|}}
+        <div class="ui {{flash.type}} message">
```

Comment:
you can add class here for adding margin around flash message.

Reply:
I tried adding margin but the only thing it does is that it stretches the pop-window.